Makefile

```diff
@@ -45,6 +46,11 @@ INTEGRATION_TEST_ARGS += -eksctl.version=$(INTEGRATION_TEST_VERSION)
 $(info will launch integration tests for Kubernetes version $(INTEGRATION_TEST_VERSION))
 endif
 
+.PHONY: github-build
+github-build:
```

Comment:
Can you move this to the workflow file?

Reply:
Yeah, with modules it should be just one line, so we won't need to wrap anything.